frontend/packages/dev-console/integration-tests/features/Helm/new-HelmCharts-Installation.feature

```diff
@@ -0,0 +1,50 @@
+Feature: Helm Chart
+    User should able to import new Helm charts and should be ablet to install newly added xhelm charts
+
+Background:
+    Given user logged into the openshift application
+    And user is at developer perspecitve
+
+@regression 
+Scenario: Add new link to fetch the new Helm Charts: HR-05-TC02
+    Given user added the new link to fetch the Helm Charts
```

Comment:
We shouldn't have this scenario in tests because we don't support changing the default helm catalog officially.

Reply:
Removed the scenario file and updated the code.